runtime/onert/backend/cpu/KernelGenerator.cc

```diff
@@ -138,26 +138,38 @@ void KernelGenerator::visit(const ir::operation::Conv2D &node)
   const auto ker_index{node.getInputs().at(Conv2D::Input::KERNEL)};
   const auto bias_index{node.getInputs().at(Conv2D::Input::BIAS)};
 
+  auto ofm_alloc = _tensor_builder->at(ofm_index).get();
+  auto ifm_alloc = _tensor_builder->at(ifm_index).get();
+  auto ker_alloc = _tensor_builder->at(ker_index).get();
+  auto bias_alloc = _tensor_builder->at(bias_index).get();
+
   const auto stride = node.param().stride;
+  const auto activation = node.param().activation;
+  const auto param_padding = node.param().padding;
+  auto fn = std::make_unique<ops::ConvolutionLayer>();
+
+  if (_ctx.at(ifm_index).info().isDynamic() || _ctx.at(ker_index).info().isDynamic())
+  {
+    const auto activation = node.param().activation;
+    auto fn = std::make_unique<ops::ConvolutionLayer>();
+    fn->configure(ifm_alloc, ker_alloc, bias_alloc, param_padding.type, param_padding.param.left,
+                  param_padding.param.right, param_padding.param.top, param_padding.param.bottom,
+                  stride.horizontal, stride.vertical, activation, ofm_alloc);
+
+    _return_fn = std::move(fn);
+    return;
+  }
```

Comment:
Is this block necessary?

Reply:
If the input shape is dynamic,it cannot perform the calculation below. So it have to stop here.

Reply:
The calculation below doesn't require many calculation. And here is in compilation time. Performance is not critical in compilation time. If it doesn't affect the work, how about removing this block for consistent and simple code?

Reply:
This block is necessary because dynamic tensor does not calculate padding.